### PR TITLE
The Details disclosure element fixes

### PR DESCRIPTION
**Pull request by dailycommit**
This is a **new bugfix**.

### Proposed solution

This fix the issues #1696 

### Testing Done

see: https://www.dropbox.com/s/ribss13dei8l0n8/bulma-box-sizing.mp4?dl=0

### CODE REVIEW

File: sass/elements/form.sass [hunk 1]
```diff
@@ -544,6 +544,8 @@ $help-size: $size-small !default
   font-size: $size-normal
   position: relative
   text-align: left
+  display: flex
```
Comment (jgthms) [inline sass/elements/form.sass]: Why the display: flex?

Comment (dailycommit) [reply sass/elements/form.sass]: I did a few more tests, I noticed that ``display: flex`` was only needed for my special case. I'm sorry for this mess 😥

**Comment by dailycommit** [comment]
Thank you Jeremy, ``display: flex`` property has been removed.

**Comment by diomed** [comment]
you forgot to merge this ⁉️

**Pull request merged**